src/components/SQForm/SQFormTextarea.tsx

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import TextField from '@material-ui/core/TextField';
 import Grid from '@material-ui/core/Grid';
-import {useFormikContext} from 'formik';
-
+import {useFormikContext, FormikProps} from 'formik';
 import {useForm} from './useForm';
+import {TextareaAutosizeProps, TextFieldProps} from '@material-ui/core'
+import BaseFieldProps from '../../types/BaseFieldProps';
 import {toKebabCase} from '../../utils';
 
+interface SQFormTextareaProps extends BaseFieldProps {
```

Comment:
Per the original propTypes, `name` and `label` are the only required props, which are included in `BaseFieldProps`, everything else here should be set to optional.

Reply:
@codyarose  field those are not required set to Optional